packages/core/src/actions.ts

```diff
@@ -157,11 +158,13 @@ export function toActivityDefinition<TContext, TEvent extends EventObject>(
  * @param eventType The event to raise.
  */
 export function raise<TContext, TEvent extends EventObject>(
-  event: Event<TEvent>
-): RaiseAction<TEvent> | SendAction<TContext, AnyEventObject, TEvent> {
-  if (!isString(event)) {
-    return send(event, { to: SpecialTargets.Internal });
+  event: Event<TEvent>,
```

Comment:
This doesn't support expressions right now. I think that we should bring it to here - otherwise not all `send`s can be refactored to `raise`